Acceptance-Tests.md

```diff
@@ -126,6 +128,114 @@ make run_tests.acceptance suite="blog-editor-tests/check-blog-editor-unable-to-p
 
 10) The test must be thoroughly tested before submitting a PR. The test can be run locally by running the following command as mentioned above or you can run the test on the CI server by pushing your code to the remote branch in your fork. The CI server will run the test and will show the result.
 
+### Console errors logging functionality in Acceptance Tests
+
+Acceptance Tests have the capability to detect console errors during CUJ's, potentially resulting in test failures. However, there are scenarios where certain console errors can be deemed acceptable and should not cause the test to fail. In order to ignore certain errors like these, you can use `ConsoleReporter.setConsoleErrorsToIgnore`, which takes in an array of error regexes to match the errors that can be acceptable. For instance, errors like `Blog Post with the given title exists already. Please use a different title.`, which occurs during the 'blog-editor-tests/try-to-publish-a-duplicate-blog-post-and-get-blocked' test, is ignored using the ConsoleReporter since it is an acceptable error in the context of the test. When passing acceptable errors like these to the ConsoleReporter, you should be specific and not use vague errors like `Failed to load resource...`, below is an example of this usage:
```

Comment:
ignore certain errors --> ignore errors

errors like --> an error like

Split "Below is an example ..." at the end into its own sentence.